CHANGELOG.md

```diff
@@ -8,15 +8,23 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ### Added
 
-- None
+- Add support for SF Symbols 2.2, 3.0, 3.1 & 3.2 (By [Steven Magdy](https://github.com/StevenMagdy))
+- Add support for explicit symbol localization (By [David Knothe](https://github.com/knothed))
+- Add documentation for available layersets & improve documentation in general. (By [Frederick Pietschmann](https://github.com/fredpi))
 
 ### Changed
 
-- None
+- Deprecate `allCases: [SFSymbol]` in favor of new `allSymbols: Set<SFSymbol>`. (By [Steven Magdy](https://github.com/StevenMagdy))
+- Transfer repository to new location: https://github.com/SFSafeSymbols/SFSafeSymbols & simplify contributions to this repository. (By [Frederick Pietschmann](https://github.com/fredpi))
+- Change the type of `SFSymbol` from `enum` to `class`, using `static let ...` instead of `case` for individual symbols. This allows for specification of custom symbols as `static let`s in `extension`s to `SFSymbol` and fixes an issue that prevented `rawValue` initialization of symbols. (By [Steven Magdy](https://github.com/StevenMagdy))
+- Improve internal code generation to allow for quick adjustments to new SF Symbols versions. (By [ddddxxx](https://github.com/ddddxxx))
+- Drop dedicated `SFSafeSymbols-Dynamic` product in the `Package.swift` in favour of a new unified `SFSafeSymbols` product with automatic selection between static / dynamic linking. (By [Steven Magdy](https://github.com/StevenMagdy))
+- Raise minimum platform requirements to 2017 platforms. (By [Steven Magdy](https://github.com/StevenMagdy))
```

Comment:
The result of that PR was just **lowering** watchOS from 6.0 to 4.0

Reply:
Oops, I should have taken a closer look 😬